src/lib/routes.js

```diff
@@ -26,23 +27,39 @@ module.exports = server => {
             next(new server.errors.InternalServerError(err));
           });
       }
-
-      next(new server.errors.BadRequestError(`Make sure the date format is '${dateFormat}' (time is optional)`));
+      next(new server.errors.BadRequestError(`Make sure the date format is "${dateFormat}" (time is optional)`));
     } else {
-      next(new server.errors.BadRequestError('Must provide a \'from\' date parameter, and optionally a \'to\' date'));
+      next(new server.errors.BadRequestError('Must provide a "from" date parameter, and optionally a "to" date'));
     }
   });
 
+  server.get('/dashboard', (req, res) => {
+    dashboardModel()
+      .then(props => res.render(LevDashboard, props))
+      .catch(err => server.log.error(err));
+  });
+
+  server.get('/dashboard/data', (req, res, next) => { // eslint-disable-line consistent-return
+    return dashboardModel()
```

Comment:
`return` can go and the esline line can go with it.

Reply:
Refactored